### PR TITLE
Load OpenOrders for MangoAccount

### PR DESCRIPTION
**Pull request by bdhobare**
#13 
Took me a while to deserialize Serum's [`OpenOrders`](https://github.com/project-serum/serum-dex/blob/master/dex/src/state.rs#L587).
- Tested with `positions.ts` and a few accounts.
- Unit tests probably needed but can be added later when more complex calculations are added.

### CODE REVIEW

File: include/mango_v3.hpp [hunk 1]
```diff
@@ -42,6 +44,15 @@ const Config DEVNET = {
     {"MNGO", "BTC", "ETH", "SOL", "SRM", "RAY", "USDT", "ADA", "FTT", "AVAX",
      "LUNA", "BNB", "MATIC", "", "", "USDC"}};
 
+const std::unordered_map<std::string, size_t> SERUM_PROGRAM_LAYOUT_VERSIONS = {
+    {"4ckmDgGdxQoPDLUkDT3vHgSAkzA3QRdNq5ywwY4sUSJn", 1},
+    {"BJ3jrUzddfuSrZHXSCxMUUQsjKEyLmuuyZebkcaFp2fg", 1},
+    {"EUqojwWA2rd19FZrzeBncJsm38Jm1hEhE3zsmX3bRc2o", 2},
+    {"9xQeWvG816bUx9EPjHmaT23yvVM2ZWbrrpZb9PusVFin", 3}};
+size_t getLayoutVersion(std::string programId) {
```
Comment (mschneider) [inline include/mango_v3.hpp:52]: method not used

File: include/mango_v3.hpp [hunk 2]
```diff
@@ -141,8 +152,37 @@ struct MangoAccountInfo {
...
+  uint8_t padding0[5];
+  // Account Flags:
+  //  accountFlags[0] = "initialized"
+  //  accountFlags[1] = "market"
+  //  accountFlags[2] = "openOrders"
+  //  accountFlags[3] = "requestQueue"
+  //  accountFlags[4] = "eventQueue"
+  //  accountFlags[5] = "bids"
+  //  accountFlags[6] = "asks"
+  std::bitset<64> accountFlags;
```
Comment (mschneider) [inline include/mango_v3.hpp:167]: enum is more elegant

File: include/mango_v3.hpp [hunk 3]
```diff
@@ -141,8 +152,37 @@ struct MangoAccountInfo {
   uint8_t padding[5];
 };
 
+// https://github.com/project-serum/serum-dex/blob/master/dex/src/state.rs#L587
+#pragma pack(1)
+struct OpenOrders {
```
Comment (mschneider) [inline include/mango_v3.hpp:157]: it's a serum struct, should be defined in a serum_v3 header

File: include/mango_v3.hpp (hunk 1 above)
Comment (bdhobare) [reply include/mango_v3.hpp:52]: Forgot to delete, good catch, I was originally planning to have `OpenOrdersV1` and `OpenOrdersV2` according to the serum pk used, as is in the js sdk, but that's unnecessary now.

**Comment by bdhobare** [comment]
Closing because of #31

**Pull request closed without merging**